go/vt/vtgate/endtoend/user_test.go

```diff
@@ -155,4 +156,59 @@ func TestUsers(t *testing.T) {
 	if got, want := warnings, uint16(1); got != want {
 		t.Errorf("select:\n%d want\n%d", got, want)
 	}
+
+	//deleting all records from vt_user table
+	exec(t, conn, "delete from vt_user where id > 0")
+}
+
+func TestQueryStream(t *testing.T) {
+	ctx := context.Background()
+	conn, err := mysql.Connect(ctx, &vtParams)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer conn.Close()
+
+	//Test insert
+	for i := 1; i <= 4; i++ {
+		each := strconv.Itoa(i)
+		nameQ := "test " + each
```

Comment:
what's this variable supposed to mean? It seems weird to end with uppercase Q. Can it be just called name? or nameValue?

Reply:
Done